app/cdap/components/ConnectionList/index.tsx

```diff
@@ -234,39 +237,49 @@ export default function ConnectionList() {
                     variant="body2"
                     className={classes.headerLabel}
                     ref={(element) => {
-                      refs.current[index] = element;
+                      refs.current[tabIndex] = element;
                     }}
                   >
-                    {dataForTabs[index - 1].selectedTab}
+                    {dataForTabs[tabIndex - 1].selectedTab}
                   </Typography>
                 </Box>
               );
           }
           return (
-            <Box className={classes.tabsContainerWithHeader}>
+            <Box
+              className={classes.tabsContainerWithHeader}
+              data-testid={`connection-tabs-column-${tabIndex}`}
+            >
               <Box className={classes.tabHeaders}>{headerContent}</Box>
               <ConnectionsTabs
-                tabsData={each}
+                tabsData={eachTabItem}
                 handleChange={selectedTabValueHandler}
-                value={each.selectedTab}
-                index={index}
+                value={eachTabItem.selectedTab}
+                connectionColumnIndex={tabIndex}
                 connectionId={connectionId || ''}
                 toggleLoader={(value: boolean, isError?: boolean) => toggleLoader(value, isError)}
-                setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
+                setToaster={setToaster}
               />
             </Box>
           );
         })}
       </SelectDatasetWrapper>
-
       {loading && (
         <div className={classes.loadingContainer}>
           <LoadingSVG />
         </div>
       )}
-      {isErrorOnNoWorkspace && (
-        <ErrorSnackbar handleCloseError={() => setIsErrorOnNoWorkSpace(false)} />
-      )}
+      {toaster.open && (
+        <Snackbar
+          handleCloseError={() =>
+            setToaster({
+              open: false,
+            })
+          }
+          description={toaster.message}
+          isSuccess={toaster.isSuccess}
+        />
+      )}{' '}
```

Comment:
are these spaces necessary?

Reply:
No, We have removed those spaces.

Reply:
am I missing something or are these spaces still here?